### DISCUSSION TITLE
Does Artillery support http/3 protocol?

### DISCUSSION THREAD

**Discussion by MarjuszeG** [artilleryio/artillery#1376] [Artillery]
Hello Artillery community!

My question is: if the server has `http/3` turned on, does Artillery use it or stick with typical `http/2`?
Can I check it somehow?
Is there any option to force `http/3` protocol during load tests?

Thank you for any answer :).

**Comment by hassy** [comment]
hi @MarjuszeG 👋 we don't have support for http/3 yet. We use [Got](https://github.com/sindresorhus/got) as the underlying HTTP library, so we'll need to wait for Got to add support for http/3 first.

**Reply by MarjuszeG** [reply]
Thanks a lot for the info!
